mobile/src/calendar/calendar-actions-button-group.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { TouchableHighlight, TouchableOpacity, View } from 'react-native';
+import { calendarActionsStyles, CalendarEventsColor } from './styles';
+import { CalendarActionButtonSeparator } from './calendar-action-button';
+import { AppState } from '../reducers/app.reducer';
+import { connect } from 'react-redux';
+import { IntervalsModel, DayModel, IntervalModel } from '../reducers/calendar/calendar.model';
+import { CalendarEventsType } from '../reducers/calendar/calendar-events.model';
+import { VacationActionButton } from './vacation-action-button';
+import { DayoffActionButton } from './dayoff-action-button';
+import { SickLeaveActionButton } from './sick-leave-action-button';
+
+interface ActionButtonGroupProps {
+    intervalsModel: IntervalsModel;
+    selectedCalendarDay: DayModel;
+}
+
+export class ActionsButtonGroupImpl extends Component<ActionButtonGroupProps> {
+    public render() {
+        const { vacation, dayoff, sickleave } = this.getIntervals();
+
+        return (
+            <View style={calendarActionsStyles.container}>
+                <VacationActionButton interval={vacation} />
```

Comment:
If you are trying to vertically align the buttons, then you should add one more `CalendarActionButtonSeparator` before vacation button

Reply:
I don't need separator before vacation button, there is now align in the sketch